packages/ui/package.json

```diff
@@ -122,6 +122,7 @@
     "eslint-plugin-jsx-a11y": "^6.10.2",
     "eslint-plugin-mocha": "^10.5.0",
     "eslint-plugin-no-only-tests": "^3.3.0",
+    "eslint-plugin-perfectionist": "^3.9.1",
```

Comment:
This should be installed at the top level as you include it at the top level `eslintrc`?

Reply:
Yes